templates/react/components/foo/List.js

```diff
@@ -46,9 +50,16 @@ class List extends Component {
           <tbody>
           {this.props.items.map(item =>
             <tr className={item['@id']} key={item['@id']}>
-              <td>{item['@id']}</td>
+              <td>
+                <Link to={`show/${encodeURIComponent(item['@id'])}`}>
```

Comment:
Can you merge those 3 lines in a single one (as its a code generator, the less LOC we generate, the better)